Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "deno_lint"
-version = "0.13.0"
+version = "0.14.0"
```

Comment:
We might want to wait for swc 0.59.0 and do a release tomorrow as it has some static block fixes https://github.com/swc-project/swc/commit/99c35ff9800b3d04604800a98994758b5581b5de#r55725172

Reply:
I'll just cut another one, I want to integrate rule filtering into my PR

Reply:
Ok, there will be some stuff to change because of the changes in deno doc and there are some deno graph changes going to be published once swc_ecmascript 0.60 lands.

```
error[E0432]: unresolved import `deno_doc::parser::DocFileLoader`
  --> cli\tools\doc.rs:20:5
   |
20 | use deno_doc::parser::DocFileLoader;
   |     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ no `DocFileLoader` in `parser`

error[E0603]: module `parser` is private
  --> cli\tools\doc.rs:20:15
   |
20 | use deno_doc::parser::DocFileLoader;
   |               ^^^^^^ private module
   |
note: the module `parser` is defined here
  --> C:\Users\david\.cargo\registry\src\github.com-1ecc6299db9ec823\deno_doc-0.11.0\src\lib.rs:21:1
   |
21 | mod parser;
   | ^^^^^^^^^^^
```